osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -91,6 +92,27 @@ protected override void LoadComplete()
                     skins.SelectRandomSkin();
                 }
             });
+
+            skins.PresentSkinsImport += presentSkinsImport;
+        }
+
+        private void presentSkinsImport(IEnumerable<Live<SkinInfo>> importedSkins)
+        {
+            SettingsOverlay settings = this.FindClosestParent<SettingsOverlay>();
+
+            switch (importedSkins.Count())
+            {
+                case 1:
```

Comment:
does this need to be a switch? why not an early return if statement?

Reply:
It can be an if statement. Does it bring any performance benefits compare to switch?

Reply:
not really about performance afaik, more so its just mostly preferred here when there is only two possibilities

Reply:
Can you refer to any code that does this?

Reply:
We don't really use the `case >` sort of syntax in this repository. I'm not sure I mind it but it hasn't generally caught on. And with two case labels it's pretty verbose over a plain `if` anyway, so I'd also like to see this changed.

Reply:
Early return here provides non-null variable which is good idea from my perspective, for this PR I don't see any reason to use early return. Will change to if-elseif tho